autoendpoint/src/routers/stub/settings.rs

```diff
@@ -0,0 +1,25 @@
+/// Settings for `StubRouter`
+#[derive(Clone, Debug, serde::Deserialize)]
+#[serde(default)]
+#[serde(deny_unknown_fields)]
+pub struct StubSettings {
+    pub url: String,
+    #[serde(rename = "credentials")]
+    pub server_credentials: String,
+}
+
+/// Credential information for each application
+#[derive(Clone, Debug, serde::Deserialize, serde::Serialize)]
+pub struct StubServerSettings {
+    pub project_id: String,
+    pub error: String,
+}
```

Comment:
The creds aren't used at all, kill?
```suggestion
}

```

Reply:
Ah! Thanks, forgot to make the change to router.